proxy/coco/src/peer/run_state.rs

```diff
@@ -74,21 +84,27 @@ impl MaybeFrom<&Input> for Event {
             Input::Announce(input::Announce::Succeeded(updates)) => {
                 Some(Self::Announced(updates.clone()))
             },
-            Input::Peer(event) => match event {
-                PeerEvent::GossipFetch(FetchInfo {
```

Comment:
There's no `PeerEvent` anymore so we moved this logic to here https://github.com/radicle-dev/radicle-upstream/pull/1463/files#diff-463e58bd670bb12315d14a2f26fbfc88e97a22551753a25ecd20fe3056fe9de3R89